habitat/utils/gym_definitions.py

```diff
@@ -38,12 +38,31 @@ def _get_env_name(cfg: Config) -> Optional[str]:
     if (
         "GYM" in cfg
         and "AUTO_NAME" in cfg["GYM"]
+        and cfg["GYM"]["CLASS_NAME"] is not None
         and len(cfg["GYM"]["CLASS_NAME"]) > 1
     ):
         return cfg["GYM"]["CLASS_NAME"]
     return None
 
 
+def make_gym_from_config(config: Config) -> HabRenderWrapper:
```

Comment:
I now use this method in `construct_vector_env`. I plan on making it the entry point for using habitat-lab's gym environments.